.gitignore

```diff
@@ -3,4 +3,9 @@ __pycache__
 .hypothesis/
 build/
 reports/
-venv/
\ No newline at end of file
+venv/
```

Comment:
artifacts from the hardhat compile should be added to .gitignore

Reply:
added artifacts/ and .env to gitignore, provided a sample .env